src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,611 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+using System;
+using System.Threading.Tasks;
+
+namespace Tizen.Network.Bluetooth
+{
+    /// <summary>
+    /// This class is used to send commands from the control device (For example, headset) to the target device (For example, media player).
+    /// </summary>
+    /// <privilege> http://tizen.org/privilege/bluetooth </privilege>
+    /// <since_tizen> 8 </since_tizen>
+    public class BluetoothAvrcpControl : BluetoothProfile
+    {
+        private string _remoteAddress;
+        private TaskCompletionSource<bool> _taskForConnection;
+        private TaskCompletionSource<bool> _taskForDisconnection;
+        private bool disposed = false;
+
+        internal BluetoothAvrcpControl()
```

Comment:
How to get `BluetoothAvrcpControl` instance?

Reply:
BluetoothDevice.GetProfile() can get the instance.